chapter02/axp/ex02

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env perl
+
+use 5.014;
+use warnings;
+use Module::CoreList;
+use List::Util ();
+
+my @modules = sort keys( %{$Module::CoreList::version{5.014002}} );
+
+my $wid = List::Util::reduce { $a > length( $b ) ? $a : length( $b ) } 0, @modules;
+
+printf "%${wid}s | %s\n", "MODULE", "FIRST RELEASE";
+printf "%s-+--------------\n", '-' x $wid;
+printf "%${wid}s | %s\n", $_, Module::CoreList->first_release( $_ ) foreach @modules;
```

Comment:
Pretty output! But the task asked for the release date rather than the first release version.
```
                                         MODULE | FIRST RELEASE
------------------------------------------------+--------------
                                    AnyDBM_File | 5
                                      App::Cpan | 5.011003
                                     App::Prove | 5.010001
                              App::Prove::State | 5.010001
                      App::Prove::State::Result | 5.010001
                App::Prove::State::Result::Test | 5.010001
                               Archive::Extract | 5.009005
                                   Archive::Tar | 5.009003
                         Archive::Tar::Constant | 5.009003
                             Archive::Tar::File | 5.009003
                            Attribute::Handlers | 5.007003
                                     AutoLoader | 5
                                      AutoSplit | 5
                                              B | 5.005
                                     B::Concise | 5.006001
                                       B::Debug | 5.005
                                     B::Deparse | 5.005
                                        B::Lint | 5.005
```